gallery/gallery/lib/pages/home.dart

```diff
@@ -700,6 +700,7 @@ class _CarouselState extends State<_Carousel>
         controller: _controller,
         itemCount: widget.children.length,
         itemBuilder: (context, index) => builder(index),
+        allowImplicitScrolling: true,
```

Comment:
If i remember correctly, this flag was new and we couldn't use it yet because it wasn't in stable. Is this no longer the case?

Reply:
I wasn't sure, so I figured I'd let CI tell me

Reply:
Looks like it's in beta but not stable yet

Reply:
Ok, let's hold off on this until the next stable release then

Reply:
cool, I'll repurpose this PR